### ISSUE TITLE
Evaluate Qwen3MoE IQ2_S/IQ4_XS expert activations against Q8_K

### ISSUE THREAD

**Issue by Nando2392** [Nando2392/qx-mini-c#10]
## Objetivo

Evaluar y, sólo con golden independiente, implementar activaciones Q8_K CPU para expertos Qwen3MoE de tipos ggml 22/23 (IQ2_S/IQ4_XS) observados en layers 1 y 47.

## Baseline

- El bisect de #9 cerró layer 0 para tipos 17/18.
- Primera divergencia material actual: input layer 2.
- Layer 1: gate/up/down `22/22/23`, row bytes `656/656/408`.
- Layer 47: gate/up/down `22/22/23`, row bytes `656/656/408`.
- QX conserva fallback F32 explícito para esos tipos.

## Criterios

- [x] Verificar traits públicos ggml y vec-dot Q8_K exacto para IQ2_S (22) e IQ4_XS (23) en oracle fijado.
- [x] RED→GREEN byte/kernel por filas representativas y expertos reales.
- [x] Mismo `ffn_inp-1` para QX/llama; matriz router→weighted y `l_out-1`.
- [x] Fallback y metadata de ejecución reales; F32 sigue default.
- [x] Repetir 48-layer first-divergence, logits, greedy y perf/RSS.
- [x] Fail-closed overflow/OOB/layout/NaN/Inf.
- [x] Build, suite, smoke, wiki, Auto Research y 3 reviews sobre mismo digest.

No perseguir argmax ni extrapolar fórmulas de IQ2_XS/IQ3_XXS. El tipo 23 es IQ4_XS, no IQ3_S; la numeración se verifica contra `ggml.h` del oracle fijado.

**Comment by Nando2392** [comment]
## Implementación verificada

Entregado:

- Corregido el mapeo de tipos del oracle fijado: `22=IQ2_S`, `23=IQ4_XS`.
- Añadidos vec-dot CPU `IQ2_S × Q8_K` e `IQ4_XS × Q8_K`, verificados contra traits públicos ggml.
- Oracle ampliado con captura seleccionable `internals=N`.
- Layer 1 validada con exactamente el mismo `ffn_inp-1`: mezcla final max-abs `4.34666e-5`, RMSE `9.61547e-7`, cosine ≈ `1`.
- Metadata basada en familias realmente ejecutadas y fallback F32 explícito.
- Metadata exacta por subruta (`gate/up` y `down`) tras corregir el blocker del primer review.
- E2E ampliado con checkpoint `layer-2-input`; primera divergencia material sigue en layer 2 por amplificación, no por vec-dot sin golden.
- Greedy permanece sin paridad y F32 continúa como default.
- Benchmark warm (5 repeticiones, KV INT8): F32 `8.00310 s/token`, Q8_K `2.28223 s/token`, speedup `3.50670×`; RSS mediano difiere `4096 B`.

Evidencia local:

```text
MSVC C17 /O2 /W4: PASS
pytest: 118 passed in 431.33s
smoke_check.py: PASS
wiki lint: PASS (16 knowledge pages, 20 notes)
compileall: PASS
git diff --cached --check: PASS
Auto Research: ready_for_review
staged artifact/secret audit: PASS
reviews independientes: 3/3 PASS
```

Commit: `03093eb010d5b95ee2b0017c03caa5d2f220cfa5`
CI: https://github.com/Nando2392/qx-mini-c/actions/runs/32086879270 — SUCCESS
Digest revisado: `4dceca0e18222e0ec8c66969c242233aad371d7b4aef7bd722e944acc614b2b8`

Siguiente gate: bisectar la amplificación layer 1 → layer 2, empezando por el experto dominante 68; no reabrir los vec-dot validados sin evidencia nueva.
